server/server.js

```diff
@@ -1,37 +1,36 @@
 const express = require('express');
-const cors = require('cors');
+//const cors = require('cors');
+
 const mongoose = require('mongoose');
-// const parseImage = require('./visionAPI');
-require('dotenv').config({ path: '../.env' });
-const dbKey = process.env.dbKey;
+require('dotenv').config();
```

Comment:
the problem was the path, it returned undefined, I console logged the new code and was able to get the key

Reply:
This one I need to test out tomorrow. The path shouldn't be an issue.